db/seeds.rb

```diff
@@ -12,103 +12,120 @@
-                             categories: [curry_cat, entree_cat])
+                             categories: [curry_cat, entree_cat],
+                             image: File.open('public/PhoenixSpicyCurry.jpg'))
 
 titan          = Item.create(title: "Titan Burger",
                              description: "So big and powerful it'll descend upon your tastebuds in an epic clash",
                              price: 9.25,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/TitanBurger.jpg'))
 
 mermaid        = Item.create(title: "Filet of Mermaid",
                              description: "Only wild-caught...no farming here!",
                              price: 13.00,
                              status: "retired",
-                             categories: [seafood_cat, entree_cat])
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/FiletOfMermaid.jpg'))
 
 minotaur       = Item.create(title: "NY Strip Minotaur",
                              description: "A dish served as rare as the beast itself",
                              price: 19.99,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/NYStripMinotaur.jpg'))
 
 scylla         = Item.create(title: "Scyllamari",
                              description: "Trust us when we say this dish isn't 'Charybian'",
                              price: 17.50,
                              status: "retired",
-                             categories: [seafood_cat, entree_cat])
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/Scyllamari.jpg'))
 
 griffin_wings  = Item.create(title: "Buffalo Griffin Wings",
                              description: "These little wings pack an especially powerful--maybe even majestic--punch",
                              price: 6.00,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloGriffinWings.jpg'))
 
 pegasus_wings  = Item.create(title: "Buffalo Pegasus Wings",
                              description: "One little bit will set your tastebuds aflight",
                              price: 6.00,
                              status: "retired",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloPegasusWings.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.